ci

```diff
@@ -7,6 +7,57 @@ This is the Everest CI script.
 ... environment: $(uname -a)
 INFO
 
+# set list of Everest Slack Users - added name for readability and way to associate who is who just in case want to use name in future
```

Comment:
For readability, we could store this array in a separate file (in the same repository), where each line contains exactly one username. This would simplify several of your functions below.